config/initializers/_config.rb

```diff
@@ -26,6 +26,7 @@ module Convection
     datadog_trace_agent_hostname: ENV["DATADOG_TRACE_AGENT_HOSTNAME"],
     datadog_debug: ENV["DATADOG_DEBUG"] == "true",
     debug_email_address: ENV["DEBUG_EMAIL_ADDRESS"] || "sarah@artsymail.com",
+    forque_url: ENV["FORQUE_URL"] || "https://tools.artsy.net",
```

Comment:
Don't we usually use staging for these defaults? Not a blocking comment.

Reply:
Fair. I'd copied this [from Vibrations](https://github.com/artsy/vibrations/blob/d3612705f81985bf37c4c84470c089d34a90a5f5/config/initializers/_config.rb#L8) which defaults to production. This file is itself inconsistent, but I'll push a change defaulting all to staging-appropriate values. The updated migration is now:


```
hokusai production env set ARTSY_CMS_URL=https://cms.artsy.net FORQUE_URL=https://tools.artsy.net
```